docs/src/docs/app.tsx

```diff
@@ -67,300 +66,384 @@ const markdownPreviewContent = changelogContent.reduce((acc, item, index) => {
   return `${acc}${separator}${item.content}\n\n`
 }, '')
 
+const people = [
+  {
+    name: 'Magnus',
+    image: magnus,
+    github: 'https://github.com/magnusohlin',
+    slack: '',
+    linkedin: 'https://www.linkedin.com/in/magnus-ohlin-4512652a/',
+  },
+  {
+    name: 'Sebastian',
+    image: sebbe,
+    github: 'https://github.com/SebCodesTheWeb',
+    slack: '',
+    linkedin: 'https://www.linkedin.com/in/sebastian-delgado-frontdev/',
+  },
+  {
+    name: 'Paolo',
+    image: paolo,
+    github: 'https://github.com/PaoloGiraudi',
+    slack: '',
+    linkedin: 'https://www.linkedin.com/in/paologiraudi/',
+  },
+  {
+    name: 'Hanna',
+    image: hanna,
+    github: 'https://github.com/HannaYankouskaya',
+    slack: 'https://mediatool.slack.com',
+    linkedin: '',
+  },
+  {
+    name: 'Ilija',
+    image: iliya,
+    github: 'https://github.com/ilija-',
+    slack: '',
+    linkedin: 'https://www.linkedin.com/in/ilijab/',
+  },
+]
+
 const DocsApp = () => (
-  <Box
-    w="100%"
-    h="100%"
-    p={ 12 }
-    pt="0"
-    overflowY="auto"
-    overflowX="hidden"
-    overscrollBehavior="contain"
-    bgColor="background.default"
-    maxH="90vh"
-  >
-    <HStack
-      sx={ { bgGradient: 'linear(to-b, teal.800, mediatoolBlue.700)' } }
-      borderRadius="md"
-      justifyContent="space-around"
-      height="2xl"
-    >
-      <Flex gap={ 10 }>
-        <Box alignSelf="flex-start" p={ 20 } justifyContent="space-around">
-          <Fade in={ true } enterDelay={ 800 }>
-            <Capitalized
-              sx={ { fontSize: '64px', fontWeight: 'regular', color: 'white' } }
+  <VStack spacing={ 8 } alignItems="center">
+    <Fade in={ true } enterDelay={ 200 }>
+      <Grid
+        templateColumns={ { base: '1fr', md: '1fr 1fr' } }
+        borderRadius="md"
+        height={ { base: 'auto', md: 'md' } }
+        p={ { base: 4, md: 10 } }
+        alignItems="center"
+      >
+        <GridItem
+          display="flex"
+          justifyContent={ { base: 'start', md: 'flex-end' } }
+          alignItems="center"
+          order={ { base: 1, md: 2 } }
+        >
+          <Icon
+            as={ NorthlightLogoXs }
+            boxSize={ { base: '6rem', md: '24rem' } }
+            alignSelf="center"
+            viewBox="0 -2 22 22"
+          />
+        </GridItem>
+        <GridItem order={ { base: 2, md: 1 } }>
+          <Box color="white">
```

Comment:
I think you can remove color="white", idk think it does anything because it is overriden by the children